.github/workflows/release.yml

```diff
@@ -0,0 +1,290 @@
+name: release
+
+on:
+  push:
+    branches:
+    - release/v*
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    outputs:
+      release_pr: ${{ steps.release-please.outputs.pr }}
+      release_version: ${{ steps.release-please.outputs.version }}
+    permissions:
+      contents: write
+      id-token: write
+      issues: write
+      pull-requests: write
+      actions: read
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0 https://github.com/actions/checkout
+        with:
+          fetch-depth: 0
+      - uses: googleapis/release-please-action@c2a5a2bd6a758a0937f1ddb1e8950609867ed15c # v4.3.0 https://github.com/googleapis/release-please-action/commits/main/
+        name: release-please
+        id: release-please
+        with:
+          target-branch: ${{ github.ref_name }}
+          config-file: release-please-config.json
+          manifest-file: .release-please-manifest.json
+      # These run only if a release PR was opened or modified, so not when the PR is merged
```

Comment:
I'm a bit confused how this works for PRs, as the workflow only specifies `on.push.branches`. Is this something special that `release-please` can do?